diversity-lunch-be/src/test/java/de/brockhausag/diversitylunchspringboot/integrationstests/EMailControllerIT.java

```diff
@@ -95,4 +128,36 @@ public void testSendTestMail_expectedCorrectTestContentInMail() {
         Assertions.assertEquals("Hallo :)", body);
         Assertions.assertEquals(1, amount);
     }
+
+    @SneakyThrows
+    @Test
+    public void testSendTestMailToUser_withValidId_expectedOkStatus() {
+        when(this.diversityLunchMailProperties.getSender()).thenReturn("diversitylunchtest@brockhaus-ag.de");
+        ProfileEntity tmpProfileEntity = profileTestdataFactory.createEntity();
+        AccountEntity accountEntity1 = accountService.getOrCreateAccount(tmpProfileEntity.getEmail());
+        //ProfileEntity is needed, please don't remove it, although IntelliJ suggests to do so.
+        ProfileEntity profileEntity1 = profileService.createProfile(tmpProfileEntity, accountEntity1.getId()).orElseThrow();
+        long id = tmpProfileEntity.getId();
+
+        mockMvcSecurity.perform(
+                post("/mailing/sendTestMailToUser?id=" + id)
+                        .header(HttpHeaders.AUTHORIZATION, "Bearer " + profileTestdataFactory.getTokenStringFromId(accountEntity1.getUniqueName()))
+        ).andExpect(status().isOk());
```

Comment:
Either you make this a unit test and test only if DiversityLunchEMailService.sendEmail gets called once in the process or you have to check if mailhoog REALY did send the specified email.

Reply:
SendMail function has been tested in a former test. Here the correct functioning of the authorization of the controller was meant to be tested. This can't be done with a unit test afaik.

Reply:
I suppose this can be tested by a unit test, since all the functionality, of prohibiting a call from an unauthorized user, comes from the @PreAuthorize("isProfileOwner(#id)") annotation, which is also active while running unit tests.

Since this integration test doesn't do any integration of two or more systems, it is just a unit test.